src/components/DefinitionEntry.js

```diff
@@ -91,6 +103,26 @@ export default class DefinitionEntry extends React.Component {
     return licenseExpression ? <span>{licenseExpression}</span> : <span>&nbsp;</span>
   }
 
+  getRevisionUrl(definition) {
+    const coordinate = get(definition, 'coordinates')
+    switch (coordinate.provider) {
+      case 'github':
+        return ''
+      case 'npmjs':
+        return `${this.getLocationURL(definition)}/v/${coordinate.revision}`
+    }
+  }
+
+  getLocationURL(definition) {
```

Comment:
suggest a different name. "location" is too generic and redundant with "url". Perhaps `getComponentUrl` vs `getRevisionUrl`